notification_center.py

```diff
@@ -7,7 +9,7 @@
 
 SCRIPT_NAME = 'notification_center'
 SCRIPT_AUTHOR = 'Sindre Sorhus <sindresorhus@gmail.com>'
-SCRIPT_VERSION = '0.2.0'
+SCRIPT_VERSION = '0.2.1'
```

Comment:
Opss, sorry. I didn't notice this changes, I've being using this for quite sometime and realized it could be useful for other people too. I'll remove the unnecessary changes and squash.